exporters/geneva/include/opentelemetry/exporters/geneva/metrics/uuid.h

```diff
@@ -0,0 +1,528 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This header copies required code from [here](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/exporters/etw/include/opentelemetry/exporters/etw/utils.h) and [here](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/exporters/etw/include/opentelemetry/exporters/etw/uuid.h). Can be ignored during review, as it was already reviewed as part of ETW exporter.